run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd /app 
+export LD_LIBRARY_PATH=usr/lib
```

Comment:
why is this needed?

Reply:
it's invalid anyway.

Reply:
> why is this needed?

make the executable in /app/bin

Reply:
still it's invalid because these must be absolute path, always. and if you have anything `/app/usr` then it's installed wrong.